### ISSUE TITLE
Support RouteType default

### ISSUE THREAD

**Issue by sherl0cks** [rht-labs/automation-api#11]
```
{
  "timestamp": 1477596978182,
  "status": 400,
  "error": "Bad Request",
  "exception": "org.springframework.http.converter.HttpMessageNotReadableException",
  "message": "Could not read document: Can not deserialize value of type com.rhc.automation.model.Route$RouteTypeEnum from String \"default\": value not one of declared Enum instance names: [reencrypt, passthrough, edge]\n at [Source: java.io.PushbackInputStream@6955e89; line: 647, column: 25] (through reference chain: com.rhc.automation.model.Engagement[\"openshift_clusters\"]->java.util.ArrayList[0]->com.rhc.automation.model.OpenShiftCluster[\"openshift_resources\"]->com.rhc.automation.model.OpenShiftResources[\"projects\"]->java.util.ArrayList[6]->com.rhc.automation.model.Project[\"apps\"]->java.util.ArrayList[1]->com.rhc.automation.model.Application[\"routes\"]->java.util.ArrayList[0]->com.rhc.automation.model.Route[\"route_type\"]); nested exception is com.fasterxml.jackson.databind.exc.InvalidFormatException: Can not deserialize value of type com.rhc.automation.model.Route$RouteTypeEnum from String \"default\": value not one of declared Enum instance names: [reencrypt, passthrough, edge]\n at [Source: java.io.PushbackInputStream@6955e89; line: 647, column: 25] (through reference chain: com.rhc.automation.model.Engagement[\"openshift_clusters\"]->java.util.ArrayList[0]->com.rhc.automation.model.OpenShiftCluster[\"openshift_resources\"]->com.rhc.automation.model.OpenShiftResources[\"projects\"]->java.util.ArrayList[6]->com.rhc.automation.model.Project[\"apps\"]->java.util.ArrayList[1]->com.rhc.automation.model.Application[\"routes\"]->java.util.ArrayList[0]->com.rhc.automation.model.Route[\"route_type\"])",
  "path": "/engagements"
}
```

**Comment by mcanoy** [comment]
doesn't look like I have close capability but this was resolved by PR #12

**Comment by mcanoy** [comment]
@sherl0cks ^^ Shall we close this?
